rhel7/checks/oval/service_all_disabled.xml

```diff
@@ -0,0 +1,31 @@
+<def-group>
+  <definition class="compliance" id="service_all_disabled" version="1">
+    <metadata>
+      <title>All services disabled</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>Check if all services, except for systemd-* getty* and autorv*, are disabled</description>
```

Comment:
Check if all services, except for systemd, getty, and autorv services, are disabled